lib/client.ts

```diff
@@ -31,41 +24,41 @@ interface SDKResponse {
   request: any;
 }
 
-const notNil = compose(
-  not,
-  isNil
-)
+const notNil = (v: any) => !(typeof v === 'undefined' || v === null || ((v.constructor === String || v instanceof String) && v.trim() === ''));
+const TOKENS_REG_EX = new RegExp(/\/{(.*?)}/g);
+const replaceParams = (str: string, params: any) => str.replace(TOKENS_REG_EX, (_, p) => params[p] ? `/${params[p]}` : '/');
 
 function defaultFormatter(response: Response) {
-  return response.json();
+  if (response.body !== undefined && response.body !== null) {
+    return response.json();
+  }
+  return Promise.resolve({});
 }
 
 const baseRequest = (defaults: RequestInit): Function => (url: string, config: RequestInit, reqConfig?: RequestConfig): Function => {
-  const headers = new Headers()
-  headers.append('Accept', 'application/json')
-  headers.append('Content-Type', 'application/json')
+  const headers = new Headers();
+  headers.append('Accept', 'application/json');
+  headers.append('Content-Type', 'application/json');
+  headers.append('X-Procore-Sdk-Version', sdkVersionHeader);
```

Comment:
Let's update this to `Procore-Sdk-Version`. I've updated https://procoretech.atlassian.net/wiki/spaces/DEV/pages/1458995266/Procore%2BREST%2BAPI%2BRequirements#SHOULD-use-only-the-specified-proprietary-Procore-headers-Rest%2FV2%2B to reflect this change as well.

Reply:
Update the readme as well.